Cup_of_join.py

```diff
@@ -0,0 +1,14 @@
+# region Cup of join
+
+def Cup_of_join(*args, sep='-'):

```

Comment:
Following conventions - 
https://flaviocopes.com/python-pep8-style-guide/ 
https://peps.python.org/pep-0008/
what are peps and what is pep 8

'cup_of_join' instead of 'Cup_of_join'